library/src/main/java/bio/terra/landingzone/service/iam/LandingZoneSamClient.java

```diff
@@ -34,7 +34,7 @@ public LandingZoneSamClient(
 
   private ApiClient getApiClient(String accessToken) {
     ApiClient apiClient = getApiClient();
-    apiClient.setAccessToken(accessToken);
+    apiClient.setBearerToken(accessToken);
```

Comment:
See https://github.com/broadinstitute/sam/pull/1430/ -- this changed with the new auth security schemes in Sam swagger